cpp/api/include/trtorch/trtorch.h

```diff
@@ -517,14 +517,15 @@ TRTORCH_API std::string ConvertGraphToTRTEngine(
  * in a TorchScript module
  *
  * @param engine: std::string - Pre-built serialized TensorRT engine
+ * @param info: trtorch::CompileSpec - Compilation settings
  *
  * Takes a pre-built serialized TensorRT engine and embeds it in a TorchScript
  * module. Registers execution of the engine as the forward method of the module
  * Forward is defined as: forward(Tensor[]) -> Tensor[]
  *
  * @return: A new module trageting a TensorRT engine
  */
-TRTORCH_API torch::jit::Module EmbedEngineInNewModule(const std::string& engine);
+TRTORCH_API torch::jit::Module EmbedEngineInNewModule(const std::string& engine, CompileSpec info);
```

Comment:
We should just do CompileSpec::Device since the rest is unused, and allow this to work without the device parameter where we just select a device that fits the settings for the engine

Reply:
ok, that can be done